stdlib/asm/math/u64.masm

```diff
@@ -326,6 +326,65 @@ export.checked_eqz
     and
 end
 
+# Performs greater-than comparison of two unsigned 64 bit integers and returns min value.
```

Comment:
nit: prefer this to be "Compares two unsigned 64 bit integers and drop the larger one from the stack."